docs/frequently_asked_questions.rst

```diff
@@ -35,38 +35,46 @@ If you don't find an answer to your question here, feel free to file a `Issues`_
 
 
-**5. Is GraphScope a graph database?**
+5. Is GraphScope a graph database?
 
     GraphScope is not a graph database, however there is a persistent storage component that can serve as database inside GraphScope called `graphscope-store <https://graphscope.io/docs/persistent_graph_store.html>`_.
 
 
-**6. What's the compatibility of Gremlin in GraphScope?**
+6. What's the compatibility of Gremlin in GraphScope?
 
     GraphScope supports most querying operators in Gremlin. You may check the compatibility in this `link <https://graphscope.io/docs/interactive_engine.html#unsupported-features>`_.
 
 
-**7. The system seems get stuck, what are the possible reasons?**
+7. The system seems get stuck, what are the possible reasons?
 
     If GraphScope seems to get stuck, the possible cause might be:
 
     - In the session launching stage, the most cases are waiting for Pods ready. The time consuming may be caused by a poor network connection during pulling image, or by failing to acquire the requested resources to launch a session.
     - In the graph loading stage, it is time consuming to load and build a large graph.
     - When running a user-defined or built-in analytical algorithm, it may takes time to compile the algorithm over the loaded graph.
 
-**8. Why `No such file or directory` error when loading graph?**
+8. Why `No such file or directory` error when loading graph?
 
     This mostly occur when you are deploying GraphScope in a Kubernetes cluster, the file must be visible to the ``engnine`` Pod of GraphScope. You may need to mount a volume to the Pods or use cloud storage providers.
 
     Specifically, if your cluster is deployed with `kind <https://kind.sigs.k8s.io>`_, you may need to setup `extra-mounts <https://kind.sigs.k8s.io/docs/user/configuration/#extra-mounts>`_ to mount your local directory to kind nodes.
 
+9. What's the relationship between ``k8s_vineyard_mem``, ``vineyard_shared_mem`` and ``k8s_engine_mem``?
 
-*I do have many other questions...*
+    - ``k8s_vineyard_mem``: The memory allocated for the vineyard container.  It stores the metadata of blobs managed by vineyard, such as the shape, id, name, and so forth. As the metadata would be much smaller than datasets, the default configuration is sufficient in most cases. It's equivalent to ``vineyard.resources.memory.requests`` and ``vineyard.resources.memory.limits`` in graphscope helm charts.
+
+    - ``vineyard_shared_mem``: The memory where the data would be loaded in. Its value needs to be adjusted according to the size of the datasets. We found that setting the value to 5 times the size of the datasets on disk is usually a reasonable value. It's equivalent to ``vineyard.shared_mem`` in graphscope helm charts.
```

Comment:
can we keep this parameter for advanced users? by default, maybe only expose `k8s_vineyard_mem` for meta and `k8s_engine_mem` for data are sufficient?

Reply:
Sounds good.